include/quda.h

```diff
@@ -1214,37 +1214,26 @@ extern "C" {
    * @param _hp_x       Array of solution spinor fields
    * @param _hp_b       Array of source spinor fields
    * @param param       Contains all metadata regarding host and device storage and solver parameters
-   * @param h_gauge     Base pointer to host gauge field (regardless of dimensionality)
-   * @param gauge_param Contains all metadata regarding host and device storage for gauge field
    */
-  void invertMultiSrcQuda(void **_hp_x, void **_hp_b, QudaInvertParam *param, void *h_gauge, QudaGaugeParam *gauge_param);
+  void invertMultiSrcQuda(void **_hp_x, void **_hp_b, QudaInvertParam *param);
```

Comment:
@hummingtree I guess this will necessitate changes to CPS: I assume CPS has split-grid plumbed in with QUDA?

Reply:
I have gone and removed the `invertMultiSrcStaggeredQuda` and `invertMultiSrcCloverQuda` as well as `dslashMultiSrcStaggeredQuda` and`dslashMultiSrcCloverQuda`

Reply:
It has, but I don't think it is actually used.